pynecone/components/graphing/victory.py

```diff
@@ -0,0 +1,567 @@
+"""Victory graphing components."""
+
+from typing import Any, Dict, Union, List, Optional
+
+from pynecone.components.component import Component
+from pynecone.components.tags import Tag
+from pynecone.var import Var
+
+
+def format_xy(x: List, y: List) -> List:
+    """Format x and y data.
+
+    Args:
+        x: The x values.
+        y: The y values.
+
+    Returns:
+        The formatted data.
+
+    Raises:
+        ValueError: If x and y are not the same length.
+    """
+    if len(x) != len(y):
+        raise ValueError("x and y must be the same length")
+    return [{"x": x[i], "y": y[i]} for i in range(len(x))]
+
+
+def format_line(x: List, y: List) -> List:
+    """Format line data.
+
+    Args:
+        x: The x values.
+        y: The y values.
+
+    Returns:
+        The formatted data.
+    """
+    return format_xy(x, y)
+
+
+def format_scatter(x: List, y: List, amount: Optional[List] = None) -> List:
+    """Format scatter data.
+
+    Args:
+        x: The x values.
+        y: The y values.
+        amount: The amount of each point.
+
+    Returns:
+        The formatted data.
+
+    Raises:
+        ValueError: If x and y are not the same length.
+    """
+    if x is None or y is None:
+        raise ValueError("x and y must be provided")
+
+    if amount is None:
+        return format_xy(x, y)
+    else:
```

Comment:
Don't need the `else` here since the `if` case returns. (Similarly in the other functions below)